estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api, exceptions, _
+from datetime import datetime
+from odoo.exceptions import ValidationError
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Estate Property Offer"
+    _order = "price desc"
+    _sql_constraints = [('check_price', 'CHECK (price > 0)', 'The offer price must be positive.')]
+
+    price = fields.Float('Price')
+    status = fields.Selection(string='Status', selection=[('accepted', 'Accepted'), ('refused', 'Refused')],
+                              copy=False)
+    partner_id = fields.Many2one('res.partner', string='Partner', required=True)
+    property_id = fields.Many2one('estate.property', string='Estate Property', required=True)
+    validity = fields.Integer('Validity', default='7')
+    date_deadline = fields.Date(string='Date Deadline', compute='_compute_date_deadline',
+                                inverse='_inverse_date_deadline')
+
+    @api.depends('validity')
+    def _compute_date_deadline(self):
+        #print('Hello compute')
+        for record in self:
```

Comment:
try to avoid generic name like record, and more specific ones like 'offer' here so that it's more readable